src/core/Akka.Remote/Transport/DotNetty/BatchWriter.cs

```diff
@@ -114,72 +121,66 @@ public override Task WriteAsync(IChannelHandlerContext context, object message)
              * across the network.
              */
             var write = base.WriteAsync(context, message);
-            if (Settings.EnableBatching)
```

Comment:
Removed the `Settings.EnableBatching` checks - this stage never runs if this is false.